src/store-api/src/region_request.rs

```diff
@@ -60,6 +60,7 @@ pub enum RegionRequest {
     Compact(RegionCompactRequest),
     Truncate(RegionTruncateRequest),
     Catchup(RegionCatchupRequest),
+    Sequences(RegionSequencesRequest),
```

Comment:
I don't think we need to add the `Sequences` request to the `RegionRequest`. We can add a new API to the `RegionEngine` to acquire sequences. Also we don't have to define a proto for this.